src/Cake.Frosting/CakeHost.cs

```diff
@@ -98,7 +98,7 @@ public int Run()
 
                 // Get the command and execute.
                 var command = _commandFactory.GetCommand(_options);
-                var result = command.Execute(_engine, _options);
+                var result = command.ExecuteAsync(_engine, _options).GetAwaiter().GetResult();
```

Comment:
I'm not sure of the difference, but the ASP.NET Identity team have an [`AsyncHelper`](https://github.com/aspnet/AspNetIdentity/blob/4874623fb8cdaeeded92891af5017d8480fd014a/src/Microsoft.AspNet.Identity.Core/AsyncHelper.cs) class to run async methods synchronously.

If you think it'd be better, I can push a change with that.
If you know the difference, I'd also love to hear about it 😄

Reply:
`.GetAwaiter().GetResult()` is fine.